### PR TITLE
Fix Blog Functionality: add "Content" field + remove "Link" field

### PR DESCRIPTION
**Pull request by tahazaryab**
# [CLICKUP TICKET LINK](https://app.clickup.com/t/860q2ck6r)

## Changes

- Remove "Link" field from blog section
- Add "Content" field to blog section

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing

- Updated tests to account for the removal of "link"
- All functionality should still be the same
   - Can still add, delete, edit blog posts without any errors
   - Should be a validation error if "content" is empty

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation/READ-ME/UMLS (if applicable)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by jennchenn** [comment]
Task linked: [CU-860q2ck6r Add "content" to blog posts](https://app.clickup.com/t/860q2ck6r)

File: src/frontend/pages/blogs/edit-a-blog/EditBlog.js [hunk 1]
```diff
@@ -226,12 +226,15 @@ const EditBlog = ({ add }) => {
...
+        <FormContainer title="Content (required)">
+          <BlogInfo>
+            <TextInput
+              value={content}
+              onChange={setContent}
+              placeholder=""
+              multiLine={true}
+            />
+          </BlogInfo>
         </FormContainer>
```
Comment (Samanthatb1) [inline src/frontend/pages/blogs/edit-a-blog/EditBlog.js:238]: could you place the content above the date field?

**Review by Samanthatb1 — COMMENTED** [review]
Were you able to run the integration tests?

They're not passing on my end but it could be a local issue. let me know

the steps are
`
npm run docker:test
npm run test:migrate:latest
npm run test:seed
npm run test:integration
`

File: src/frontend/pages/blogs/edit-a-blog/EditBlog.js (hunk 1 above)
Comment (tahazaryab) [reply src/frontend/pages/blogs/edit-a-blog/EditBlog.js]: updated

**Comment by tahazaryab** [comment]
> @Samanthatb1 the integration tests were already failing for me prior to this change (e.g. in a diff branch like `main`), so I don't think this affected them, can you confirm?

**Review by Samanthatb1 — APPROVED** [review]

**Pull request merged**